api/admin.go

```diff
@@ -66,15 +52,11 @@ func postLogLevelHandler(logLevel *slog.LevelVar) http.HandlerFunc {
 		case "crit":
 			logLevel.Set(log.LevelCrit)
 		default:
-			writeError(w, http.StatusBadRequest, "Invalid verbosity level")
-			return
+			return utils.BadRequest(errors.New("Invalid verbosity level"))
```

Comment:
Should we have JSON and plain text responses ? 
Perhaps it would be simpler to only have json payloads ?

Reply:
more consistent for sure

Reply:
It's better to remain consistency.